brainscore_language/plugins/sg_tse/benchmark.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from brainscore_core.benchmarks import BenchmarkBase
+from brainscore_language import load_metric, benchmarks
+from brainscore_language.plugins.sg_tse.data import user_def_params
+from brainscore_language.models.huggingface import HuggingfaceSubject
+from brainscore_language.artificial_subject import ArtificialSubject
+from brainscore_language.plugins.sg_tse import compute_surprisals, evaluate
+
+logger = logging.getLogger(__name__)
+
+BIBTEX = """MIT Psycholinguistics lab; SyntaxGym, LM-Zoo; ,
+  title={SyntaxGym Targeted Syntactic Evaluations},
+  author={MIT Pscyho-linguistics Lab, Jim Neidhoefer},
+  conference={N/A},
+  url={https://syntaxgym.org/},
+  year={2022}
+}"""
+
+class SyntaxGymTargetedSyntacticEval(BenchmarkBase):
+    def __init__(self):
+        super(SyntaxGymTargetedSyntacticEval, self).__init__(
+            identifier=user_def_params.bench_id(),
```

Comment:
where do the `user_def_params` come from? If I understand this right, we should instantiate all possible benchmarks here, rather than allowing for some configuration. Submitted benchmarks are not supposed to be flexible.

Reply:
user_def_params has been removed.